pom.xml

```diff
@@ -89,15 +89,14 @@
             <version>5.7.2</version>
         </dependency>
         <dependency>
-<<<<<<< Updated upstream
             <groupId>nz.net.ultraq.thymeleaf</groupId>
             <artifactId>thymeleaf-layout-dialect</artifactId>
             <version>2.3.0</version>
-=======
+        </dependency>
+        <dependency>
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-search-orm</artifactId>
             <version>5.11.1.Final</version>
->>>>>>> Stashed changes
```

Comment:
How is there a commit that had this in it?

Reply:
This has been fixed a long time ago

Reply:
None of the commits should have this, do an interactive rebase and remove it, as all commits need to build.

Reply:
should be fixed now.